### PR TITLE
Update Fluvio client README and front README

### PR DESCRIPTION
**Pull request by nicholastmosher**
- Adds badges to `fluvio` README so that they appear on crates.io
- Centers badges on front README and makes badge styles consistent
- Update README wording to emphasize "real-time applications"
- Add quick links to send people to the website

[Rendered](https://github.com/infinyon/fluvio/blob/ff43abf9efa4e73d295df0d85106550fb0142453/README.md)

### CODE REVIEW

**Review by simlay — APPROVED** [review]
LGTM

**Comment by sehz** [comment]
rebase

File: README.md [hunk 1]
```diff
@@ -1,56 +1,57 @@
...
 ## Release Status
-Fluvio is at Alpha and should be suitable for lab environment. APIs, Schema, CLI, and Services are continually evolving and subject to change before R1.
-
-
-## Documentation
-
-Please see doc at: https://www.fluvio.io
 
+Fluvio is currently in **Alpha** and is not ready to be used in production.
+Do not rely on Fluvio with any data that you would care to lose.
```
Comment (sehz) [inline README.md:46]: remove this line

File: README.md [hunk 2]
```diff
@@ -1,56 +1,56 @@
...
+<a href="https://discordapp.com/invite/bBG2dTz">
+<img src="https://img.shields.io/discord/695712741381636168.svg?logo=discord&style=flat" alt="chat" />
+</a>
+</div>
+</div>
 
-Roll out your own log aggregation, DB replication, IOT middle layer, or an event driven infrastructure in a matter of minutes. Use our expressive Node or Rust APIs to create producers/consumers and start rapid experimentation with real-time data.
+Fluvio is a high-performance distributed streaming platform that's written
+in Rust, built to make it easy to develop real-time applications. Fluvio is
+built on three core principles:
```
Comment (sehz) [inline README.md:29]: Remove 3 principles.  Will add back once we review exact wording

**Review by sehz — APPROVED** [review]

**Pull request merged**